pages/streams.rst

```diff
@@ -107,18 +107,16 @@ a given value. Get an alert when a message with the field `type` set to `securit
 **Important:** We do not recommend to run this on analyzed fields like `message` or `full_message` because it is broken down
 to terms and you might get unexpected alerts. For example a check for `security` would also alert if a message with the field
 set to `no security` is received because it was broken down to `no` and `security`. This only happens on the analyzed
-`message` and `full_message` in Graylog.
+`message` and `full_message` in Graylog. Please also take note that only a single alert is raised for this condition during the alerting interval, although multiple messages containing the given value may have been received since the last alert.
 
 What is the difference between alert callbacks and alert receivers?
 ===================================================================
 
-There are two type of actions to be triggered when an alert is fired: Alert callbacks or an email to a list of alert
-receivers.
+There are two groups of entities configuring what happens when an alert is fired: Alarm callbacks and alert receivers.
 
-Alert callbacks are single actions that are just called once. For example: The *Email Alert Callback* is triggering
-an email to exactly one receiver and the *HTTP Alert Callback* is calling a HTTP endpoint once.
+Alarm callbacks are a list of events that are being processed when an alert is triggered. There could be an arbitrary number of alarm callbacks configured here. If there is no alarm callback configured at all, a default email transport will be used to notify about the alert. If one or more alarm callbacks are configured (which might include the email alarm callback or not) then all of them are executed for every alert.
```

Comment:
This reads a bit confusing because the email transport is only used if there are receivers configured and people may wonder where the alarm would go to.